lib/iris/coord_systems.py

```diff
@@ -315,12 +315,20 @@ def xml_element(self, doc):
         return CoordSystem.xml_element(self, doc, self._pretty_attrs())
 
     def as_cartopy_crs(self):
+        globe = None
+        if self.ellipsoid is not None:
+            globe = self.ellipsoid.as_cartopy_globe()
         return ccrs.RotatedGeodetic(self.grid_north_pole_longitude,
-                                    self.grid_north_pole_latitude)
+                                    self.grid_north_pole_latitude,
+                                    globe=globe)
```

Comment:
Looks like this new globe handling has broken a couple of the existing tests in `iris.tests.test_mapping`.